pilot/pkg/xds/monitoring.go

```diff
@@ -118,10 +118,16 @@ var (
 	ldsSendErrPushes = pushes.With(typeTag.Value("lds_senderr"))
 	rdsSendErrPushes = pushes.With(typeTag.Value("rds_senderr"))
 
+	debounceTime = monitoring.NewDistribution(
+		"pilot_debounce_time",
+		"Delay in seconds between the first config enters debouncing and the merged push request is pushed into the push queue.",
+		[]float64{.01, .1, 1, 3, 5, 10, 20, 30, 40, 50, 60, 80, 100, 120},
+	)
+
 	pushContextInitTime = monitoring.NewDistribution(
 		"pilot_pushcontext_init_seconds",
 		"Total time in seconds Pilot takes to init pushContext.",
-		[]float64{.01, .1, 0.5, 1, 3, 5},
+		[]float64{.01, .1, 1, 3, 5, 10, 20, 30, 40, 50, 60, 80, 100, 120},
```

Comment:
This seems like too broad of a range... if it takes over 5s something is terribly wrong -- we don't need to distinguish 30s (terrible) vs 40s (also terrible)?

Reply:
our logging shows that debounce time is taking more than 1minute (we use default max delay 10s) and this suggests to me that init push context is taking very long blocking the next push. unfortunately we don't have the push context init data right now in production (because it's not [registered](https://github.com/istio/istio/pull/40049/files)).

this seems very wrong to you? how about i change the buckets to
`.01, .1, 1, 3, 5, 10, 20, 30` similar to all the others

Reply:
IMO metrics buckets do not need super high granularity at outlier levels. They just need to indicate "something is wrong, it is super high!" to alert to look into it with more detailed data (logs, profiles, etc).

1min seems excessively long

Reply:
That makes sense, for the debounce metric, I'm gonna use `.01, .1, 1, 3, 5, 10, 20, 30` which is the same as the other push time metrics (since 10s is pretty normal given the default max delay), for this init context one, do you suggest me to keep original?

Reply:
Yeah I think original one for init and what you mentioned for debounce is ok. thanks!

Reply:
updated